mysql-ballerina/build.gradle

```diff
@@ -247,6 +241,8 @@ task ballerinaPublish {
                     commandLine 'sh', '-c', "$distributionBinPath/bal push ${packageName}"
                 }
             }
+        } else {
```

Comment:
Shall we add the condition to check the `SNAPSHOT` version here?

Ref: https://github.com/ballerina-platform/module-ballerina-encoding/pull/80

Reply:
Fixed in e15438d